public/expenses/edit.php

```diff
@@ -19,6 +19,9 @@
 $title = 'Edit Expense';
 include public_path('layouts/header.php');
 ?>
+<head>
+    <script defer src="../js/AE.js"></script>
```

Comment:
It's better to use the `url` helper function like this [one](https://github.com/mohamed-abdul-fattah/life-system/blob/master/public/layouts/header.php#L15) instead of the relative path.